web/themes/custom/cloudy/templates/field/field--field-group-type.html.twig

```diff
@@ -0,0 +1,3 @@
+{% include '@components/badge/badge.twig' with {
```

Comment:
I noticed there's this twig template and a bunch of other config files for `field_group_type`, which appears to be unused, perhaps from a previous iteration? Would be good to remove and cleanup unless there's a place it's used that I'm not seeing.

Reply:
Added a followup story PGOV-1220 to remove the obsolete group type configs and templates.